detox/android/detox/build.gradle

```diff
@@ -21,6 +21,10 @@ def _rnNativeArtifact = rnInfo.isRN71OrHigher
             : 'com.facebook.react:react-native:+'
 
 android {
+    def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION.tokenize('.')[0].toInteger()
+    if (agpVersion >= 7) {
```

Comment:
One more question, @mikehardy. You state in the title about Gradle Plugin 8+, but here you check >= 7, could you explain, please?

Reply:
Sorry I didn't see this until just now - this wasn't just a breaking change, it was a non-backwards-compatible breaking change (that is, it will *only* work on AGP8+), but people may still use it on AGP7! So this was the only way the community came up with for it to work on old and new at the same time. Pain in the butt! 😆